sources/packages/backend/apps/db-migrations/src/migrations/1702672829350-AddColQueueConfigurationsIsActive.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AddColQueueConfigurationsIsActive1702672829350
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Add-col-queue-configurations-is-active.sql", "Queue"),
```

Comment:
Disable-atbc-response could be here so that we do not need a migration script as revert will remove the column

Reply:
The reason for having the script separately is to enable a process in the future tickets if we need to disable any other schedulers. Yes you are right, the revert is going to remove the column completely. But as per the order of the migration and revert, the rollback disable will make the is_active flag true and then remove the column in the next revert.

Reply:
Im ok with that @guru-aot . It was just a suggestion.